test/resources/microshift-process.resource

```diff
@@ -45,6 +46,11 @@ Wait For MicroShift
     # greenboot check script to let us use it even when not on a
     # greenboot host?
 
+Wait For Greenboot Health Check To Exit
```

Comment:
This is already defined at https://github.com/openshift/microshift/blob/main/test/resources/ostree-health.resource#L10

Reply:
thanks, addressed in the last commit